### PR TITLE
EN-5656: removed capn proto from project

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
removed capnp from project: 
-all schema files
-all Save/Load functions + converter functions
-updated go.mod / go.sum
-from marshal package

### CODE REVIEW

File: data/receipt/receipt_test.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ func TestReceipt_SettersAndGetters(t *testing.T) {
 	t.Parallel()
 
 	r := receipt.Receipt{}
-	assert.False(t, r.IsInterfaceNil())
+	assert.False(t, check.IfNil(&r))
```
Comment (iulianpascalau) [inline data/receipt/receipt_test.go]: useless assert as r is value not pointer

File: data/rewardTx/rewardTx_test.go [hunk 2]
```diff
@@ -1,39 +1,19 @@
...
-
-	assert.Equal(t, smrS, loadSMR)
-}
-
 func TestRewardTx_GettersAndSetters(t *testing.T) {
 	t.Parallel()
 
 	rwdTx := rewardTx.RewardTx{}
-	assert.False(t, rwdTx.IsInterfaceNil())
+	assert.False(t, check.IfNil(&rwdTx))
```
Comment (iulianpascalau) [inline data/rewardTx/rewardTx_test.go]: useless assert

File: data/smartContractResult/smartContractResult_test.go [hunk 3]
```diff
@@ -40,7 +20,7 @@ func TestSmartContractResult_SettersAndGetters(t *testing.T) {
 		GasPrice: gasPrice,
 		GasLimit: gasLimit,
 	}
-	assert.False(t, scr.IsInterfaceNil())
+	assert.False(t, check.IfNil(&scr))
```
Comment (iulianpascalau) [inline data/smartContractResult/smartContractResult_test.go]: useless assert

File: data/receipt/receipt.go [hunk 4]
```diff
@@ -1,74 +1,15 @@
...
-	dest := capnp.AutoNewReceiptCapn(seg)
-
-	value, _ := src.Value.GobEncode()
-	dest.SetValue(value)
-	dest.SetSndAddr(src.SndAddr)
-	dest.SetData(src.Data)
-	dest.SetTxHash(src.TxHash)
-
-	return dest
+	Value   *big.Int
```
Comment (BeniaminDrasovean) [inline data/receipt/receipt.go]: Put back the json tags for this struct.

File: data/smartContractResult/smartContractResult.go [hunk 5]
```diff
@@ -1,86 +1,20 @@
...
-	dest.SetNonce(src.Nonce)
-	dest.SetValue(value)
-	dest.SetRcvAddr(src.RcvAddr)
-	dest.SetSndAddr(src.SndAddr)
-	dest.SetData(src.Data)
-	dest.SetCode(src.Code)
-	dest.SetTxHash(src.TxHash)
-
-	return dest
+	Nonce    uint64
```
Comment (BeniaminDrasovean) [inline data/smartContractResult/smartContractResult.go]: Put back the json tags for this struct.

File: data/transaction/transaction.go [hunk 6]
```diff
@@ -2,95 +2,21 @@ package transaction
...
-	dest.SetValue(value)
-	dest.SetRcvAddr(src.RcvAddr)
-	dest.SetSndAddr(src.SndAddr)
-	dest.SetGasPrice(src.GasPrice)
-	dest.SetGasLimit(src.GasLimit)
-	dest.SetData(src.Data)
-	dest.SetSignature(src.Signature)
-
-	return dest
+	Nonce     uint64
```
Comment (BeniaminDrasovean) [inline data/transaction/transaction.go]: Put back the json tags for this struct.

File: data/receipt/receipt_test.go (hunk 1 above)
Comment (bogdan-rosianu) [reply data/receipt/receipt_test.go]: removed

File: data/rewardTx/rewardTx_test.go (hunk 2 above)
Comment (bogdan-rosianu) [reply data/rewardTx/rewardTx_test.go]: removed

File: data/smartContractResult/smartContractResult_test.go (hunk 3 above)
Comment (bogdan-rosianu) [reply data/smartContractResult/smartContractResult_test.go]: removed

File: data/receipt/receipt.go (hunk 4 above)
Comment (bogdan-rosianu) [reply data/receipt/receipt.go]: done

File: data/smartContractResult/smartContractResult.go (hunk 5 above)
Comment (bogdan-rosianu) [reply data/smartContractResult/smartContractResult.go]: done

File: data/transaction/transaction.go (hunk 6 above)
Comment (bogdan-rosianu) [reply data/transaction/transaction.go]: done

**Pull request merged**